hrqb/tasks/employees.py

```diff
@@ -79,6 +80,13 @@ def get_dataframe(self) -> pd.DataFrame:
         }
         return employees_df[fields.keys()].rename(columns=fields)
 
+    @HRQBTask.integrity_check
+    def check_unique_mit_ids(self, output_df: pd.DataFrame) -> None:
+        duplicate_mit_ids = len(output_df[output_df.duplicated("MIT ID", keep=False)])
+        if duplicate_mit_ids > 0:
+            message = f"Found {duplicate_mit_ids} duplicate 'MIT ID' column values"
+            raise ValueError(message)
+
```

Comment:
Very clear check and test!

Reply:
Thanks!  I know there are more to come... this was just the first absolutely lowest fruit to create.  Future ones will be more subtle, e.g. "no annual performance review if job start < 3 months before XX/YY date, and not part of ZZZ union".   

The idea is that these can grow organically as we encounter edge cases, or HR shares that tweaks are needed and we want to make sure we don't regress.